bin/setup

```diff
@@ -24,14 +24,14 @@ Dir.chdir APP_ROOT do
   ]
 
   puts "== Copying application.yml =="
-  run "cp config/application.yml.example config/application.yml"
+  run "test -e config/application.yml || cp config/application.yml.example config/application.yml"
 
   puts "== Copying logstash.conf =="
   run "cat logstash.conf.example | sed 's/path_to_repo/#{APP_ROOT.to_s.gsub('/', '\/')}/g' > logstash.conf"
 
   puts "== Copying sample certs and keys =="
-  run "cp keys/saml.key.enc.example keys/saml.key.enc"
-  run "cp certs/saml.crt.example certs/saml.crt"
+  run "test -e keys/saml.key.enc || cp -n keys/saml.key.enc.example keys/saml.key.enc"
+  run "test -e certs/saml.crt || cp -n certs/saml.crt.example certs/saml.crt"
 
```

Comment:
Can we make this change in a separate PR since it's not related to the New Relic changes?